test/distributed/cases/window/time_window.result

```diff
@@ -349,6 +353,9 @@ select _wstart, _wend, sum(col2) from sliding_window01 where ts > "2023-08-01 00
 _wstart    _wend    sum(col2)
 2023-08-01 00:00:00    2023-08-01 01:00:00    177.0
 2023-08-01 00:10:00    2023-08-01 01:10:00    151.0
+2023-08-01 00:20:00    2023-08-01 01:20:00    123.0
```

Comment:
Can you please double check if the result is correct?   Is this change saying that old result was WRONG?

Reply:
The old result is incorrect, there was a bug in the units of sliding and interval that were different before